### PR TITLE
Create create yesnielx.json

### PR DESCRIPTION
**Pull request by YesnielX**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://yesnielx.github.io

### CODE REVIEW

File: domains/yesnielx.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Portfolio Website",
+    "owner": {
+       "username": "YesnielX",
+       "email": "",
+       "discord": "yesnielx"
+    },
+
+    "record": {
+        "CNAME": "yesnielx.github.io/portfolio"
```
Comment (Stef-00012) [inline domains/yesnielx.json:10]: CNAME can not contain `/`

**Review by Stef-DP — DISMISSED** [review]

File: domains/yesnielx.json (hunk 1 above)
Comment (YesnielX) [reply domains/yesnielx.json]: Solved!

**Comment by YesnielX** [comment]
Now https://yesnielx.github.io

**Review by DEV-DIBSTER — APPROVED** [review]
LGTM.

**Pull request merged**